### DISCUSSION TITLE
Unknown Download error

### DISCUSSION THREAD

**Discussion by PatMoonB** [aclap-dev/video-downloadhelper#419] [Help and Q&A]
### Your question

Hi, I get download errors, sometimes (not every download creates an error, 80% are alright)

an example:
SideDownload error:
[tls @ 0000027676a24040] IO error: Error number -10054 occurred
    Last message repeated 1 times
[aac_adtstoasc @ 0000027676a62800] Error parsing ADTS frame header!
[mp4 @ 00000276772ad300] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input
[aost#0:1/copy @ 0000027676c0e940] Error submitting a packet to the muxer: Invalid data found when processing input
[out#0/mp4 @ 0000027676a57ec0] Error muxing a packet

when downloading (i.e.)
https://www.ardmediathek.de/video/der-sonntags-stammtisch/mit-caro-matzko-und-bijan-djir-sarai/br-fernsehen/Y3JpZDovL2JyLmRlL2Jyb2FkY2FzdFNjaGVkdWxlU2xvdC80MDM3MzY0MTg4MTNfRjIwMjNXTzAxNzY0NUEw

Is there a restart point for vdh?
What can I do to prevent such an error?
Thx in advance, Ralph

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: de
license: accepted
key (only 16 first characters): da9c3943d7d648d3
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\PT",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
I tried and it works fine at first try. Do you thinks there might be network issues?

**Comment by PatMoonB** [comment]
Thx, but try these two.
I tried more than once today, and they fail to download every time.

https://www.ardmediathek.de/video/der-sonntags-stammtisch/mit-caro-matzko-und-bijan-djir-sarai/br-fernsehen/Y3JpZDovL2JyLmRlL2Jyb2FkY2FzdFNjaGVkdWxlU2xvdC80MDM3MzY0MTg4MTNfRjIwMjNXTzAxNzY0NUEw

and

https://www.ardmediathek.de/video/der-amsterdam-krimi/der-dreck-der-anderen-der-amsterdam-krimi-7/das-erste/Y3JpZDovL2Rhc2Vyc3RlLmRlL2Rlci1hbXN0ZXJkYW0ta3JpbWkvMjAyNC0wNC0xOF8yMC0xNS1NRVNa

Ralph

Am So., 21. Apr. 2024 um 17:50 Uhr schrieb jcfrog ***@***.***

**Reply by GrampaWildWilly** [reply]
I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3447, Firefox 125.0.1 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

VDH downloaded your first video, the one with a bunch of people sitting around a table at a restaurant drinking beer & talking.  I don't know what they were talking about because I don't speak the language.  But it downloaded fine & played fine.  It was almost 1 hour long so I just sampled it at intervals & it played fine in VLC.

VDH also downloaded your second video, something about Amsterdam.  But all I got was a 1-minute video that was just a still image of a guy looking intently at a tablet computer.  There was some writing on the screen that I think was telling me that the video is not available in my country.  Again, it's a language I don't understand.  "Nicht verfügbar."  Yep.  Google Translate is telling me it's German for "not available."  But whatever it is, it did download fine & play fine.

**Reply by GrampaWildWilly** [reply]
You might want to take a look at your antivirus & firewall.  That may be one program or two.  You need to make sure that the CoApp directory -- the entire directory, all of its contents -- is permitted through both the antivirus & the firewall.  Plus, of course, the web sites have to be permitted as well.

But you are running Windows.  That means you should be using the Windows Defender that is built into the operating system.  If you are using some other antivirus/firewall, abandon it.  I'm using the Windows Defender built into Windows 11 & it has never blocked anything to do with VDH.

You might also check the parental controls in your browser.  This content does not appear to be anything that would get blocked by parental controls.  But there are browser settings (at least, in Firefox anyway) that can interfere with harmless content.  Make sure you're not accidentally blocking this content.

**Comment by PatMoonB** [comment]
I forgot to mention, that I'm working with Windows 10
[image: image.png]
and chrome
[image: image.png]

Ralph

Am So., 21. Apr. 2024 um 19:45 Uhr schrieb Say It ***@***.***>:

**Reply by PatMoonB** [reply]
Thx for your input, I run Win 10 as OS and Chroe as browser, as I mentioned. Both up-to-date, versions in my first request. vdh is also up-to-date.
Meanwhile, the first video is downloaded, but similars from the same web site are crashing, again.
The second video is still not downloading.
They are both ffrom the same german mediathek web sit).
Video One is about an hour long, two is 1 and a half. No big deal, really.

Anti-virus and and firewall are are, in my opinion, okay. I can download 0% of the filesI want. Some of them are the same length, there is no reason to to change a thing? The directory of CoApp is also checked.
I also am running the built-in Defender.

But I am getting similar errors, when downloading some filesm the rar is still 20%.
I might say that I'm using a WLAN and internet access in a nursing home for disabled persons with a guest access.
This is very slow (I don#t care) and somehow shaky, means that I'm kicked out when I don't move the cursor for a longer time than 15 minutes. This is annoying, bat I don't case, either.

That's why I'm asking for a restart/interruption point in vdh-chrome to restart a broken download.
And it would be reasonable for those temporally access-denied situations. There vdh ahs a specific handling problem ;)

Ralph

**Comment by PatMoonB** [comment]
Hi I but it in the GitHub discussion, too

Thx for your input, I run Win 10 as OS and Chroe as browser, as I
mentioned. Both up-to-date, versions in my first request. vdh is also
up-to-date.
Meanwhile, the first video is downloaded, but similars from the same web
site are crashing, again.
The second video is still not downloading.
They are both ffrom the same german mediathek web sit).
Video One is about an hour long, two is 1 and a half. No big deal, really.

Anti-virus and and firewall are are, in my opinion, okay. I can download 0%
of the filesI want. Some of them are the same length, there is no reason to
to change a thing? The directory of CoApp is also checked.
I also am running the built-in Defender.

But I am getting similar errors, when downloading some filesm the rar is
still 20%.
I might say that I'm using a WLAN and internet access in a nursing home for
disabled persons with a guest access.
This is very slow (I don#t care) and somehow shaky, means that I'm kicked
out when I don't move the cursor for a longer time than 15 minutes. This is
annoying, bat I don't case, either.

That's why I'm asking for a restart/interruption point in vdh-chrome to
restart a broken download.
And it would be reasonable for those temporally access-denied situations.
There vdh ahs a specific handling problem ;)

Ralph

Am So., 21. Apr. 2024 um 23:44 Uhr schrieb GrampaWildWilly <
***@***.***>:
